src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/DraftApprenticeshipController.cs

```diff
@@ -773,5 +788,62 @@ private async Task<ValidateUlnOverlapOnStartDateQueryResult> HasStartDateOverlap
                 model.EndDate.Date.Value.ToString("dd-MM-yyyy")
             );
         }
+
+        private async Task<IActionResult> HandleLearnerDataSync(EditDraftApprenticeshipViewModel model)
+        {
+            try
+            {
+                var response = await _outerApiService.SyncLearnerData(model.ProviderId, model.CohortId.Value, model.DraftApprenticeshipId.Value);
+                
+                if (response.Success)
+                {
+                    TempData.Put(LearnerDataSyncDraftApprenticeshipKey, response.UpdatedDraftApprenticeship);
+                    TempData[LearnerDataSyncSuccessKey] = LearnerDataSyncSuccessMessage;
+                }
```

Comment:
I understood we were no longer going to be using TempData[] and we were going to use RedisCache?